internal/querycoord/index_checker.go

```diff
@@ -118,9 +104,7 @@ func (ic *IndexChecker) reloadFromKV() error {
 		}
 		validHandoffReq, _ := ic.verifyHandoffReqValid(segmentInfo)
 		if validHandoffReq && Params.QueryCoordCfg.AutoHandoff {
-			// push the req to handoffReqChan and then wait to load after index created
-			// in case handoffReqChan is full, and block start process
-			go ic.enqueueHandoffReq(segmentInfo)
+			go ic.handleHandoffRequest(segmentInfo)
```

Comment:
If querycoord panics, will there be a goroutine leak?

Reply:
I think the context in the index checker will terminate this goroutine. 
I am not sure. WDYT @congqixia

Reply:
@xige-16 can not quite get the point. All goroutines are terminated after querycoord panics without recover.